microshift_support/microshift-sos-report.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="microshift-sos-report"]
+= MicroShift sos report
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-sos-report
+
+toc::[]
+
+[role="_abstract"]
+`sos` is a tool you can use to collect troubleshooting information about a host. An `sos report` will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.
+
+:FeatureName: MicroShift
+include::snippets/microshift-tech-preview-snip.adoc[leveloffset=+1]
+
+include::modules/microshift-about-sos-reports.adoc[leveloffset=+1]
+include::modules/microshift-gathering-sos-report.adoc[leveloffset=+1]
+include::modules/microshift-providing-sos-report-support.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_microshift-sos-report"]
+== Additional resources
+* link:https://access.redhat.com/solutions/2112[How to provide files to Red Hat Support (vmcore, rhev logcollector, sosreports, heap dumps, log files, etc.]
```

Comment:
We could consider linking to https://access.redhat.com/solutions/3592 here, too, for more details on sos report.